Update live site to latest

mmorey opened the pull request.

mmorey committed:
Fixed valtech hyperlink.

mmorey committed:
Merge pull request #87 from valtech-sd/matt/valtech-link

Fixed valtech hyperlink.

Commit:
- removed local panthenol and ylangylang pngs
- removed hard-coded img assets in scenario1, 2 and 3 html files
- removed a few logs
- created utils folder and buildImgAssets function
- calling buildImgAssets when contentstack data has loaded for scenario1, 2 and 3

Commit:
- renaming imported scripts to index.js for each scenario
- putting index scripts at end of body to ensure DOM ids exist
- renamed/repurposed util function getAssetsURLs exclusively for pulling URL strings from data
- creating dynamic Img assets in scenario1
- creating dynamic href for serum pngs in scenario1
- removed fixed serum hrefs from scenerio1

Commit:
- matched Svante's design for letter and A4 download links
- added github and contact us links
- styled with css
- using download_icon and github_logo svg files

mmorey committed:
Merge pull request #88 from valtech-sd/jack/79_dynamic-image-elements

Jack/79 dynamic image elements

mmorey committed:
Merge pull request #89 from valtech-sd/jack/83_added-landing-links

Added github and contact links, updated download links to Svante's design

Commit:
- updated body-text mixin in scenario1, 2, and 3 with reduced width and removed wrapCount
- extended text length in data_sources to experiment with dynamic lengths
- updated numbered-text component to dynamically set the wrap-count based on body length

mmorey committed:
Merge pull request #90 from valtech-sd/jack/78_dynamic-text

text body lengths now determine the wrap-count for more dynamic text layout

Commit:
- updated parcel build script to include a double dot in the public-url var
- build

Commit:
- using spyglass logo as a 'home' button in top center
- added createNavLinks util to build nav link depending on current page
- added detectDesktop util to determine whether to show alert pop-up
- updated main.css to style nav links
- creating nav links and detecting desktop in scenarios1, 2, and 3

Commit:
css tweakies

Commit:
Added comments on updated build process to the README.

Commit:
Built, and fixed broken image link in CSS.

Commit:
Add fallback image if ingredient image is missing.

jasonalderman-valtech reviewed (changes requested):
There appears to be an issue (after running `npm run build` and correcting the links) where the dynamic images are not working and all images in Scenario 2 (for example) appear as black boxes. 

I started a branch for this which you can use (...because I thought it might simply be the lack of a fallback image when the `ingredientURLs[domId]` was `undefined` — it wasn't). 
[jack/91_fix-dev-dynamic-images](https://github.com/valtech-sd/spyglass/tree/jack/91_fix-dev-dynamic-images)

Commit:
- added makePanel util

Commit:
- added marker events util to add custom lost and found callbacks for each scenario
- added 'personal' entry in the ratings personalized data field in data_sources
- keeping track of the found product ID in scenario3
- created saveReviewData function in tab-menu to store personal review
- calling saveReviewData from 'tilt-forward' callback
- changed a few lets to consts

Commit:
- changed public-url parcel arg to hopefully fix link paths

Commit:
new build

Commit:
- removed some logs
- added another check for image_url in getAssetURLs function
- created a buildDynamicAsset util
- added 'anonymous' cross origin setting to fix image loading
- added local generic.png as an icon fallback
- changed icons in contentstack to remove power of 2 warnings

Commit:
resolved merge conflicts with 47_scenario-switching

Commit:
Merge branch 'jack/81_store-review-data' into jack/47_scenario-switching

* jack/81_store-review-data:
  - added marker events util to add custom lost and found callbacks for each scenario - added 'personal' entry in the ratings personalized data field in data_sources - keeping track of the found product ID in scenario3 - created saveReviewData function in tab-menu to store personal review - calling saveReviewData from 'tilt-forward' callback - changed a few lets to consts
  - added makePanel util

# Conflicts:
#	src/scenario1/index.js
#	src/scenario2/index.js
#	src/scenario3/index.js

Commit:
Merge branch 'jack/47_scenario-switching' into jack/91_fix-dev-dynamic-images

* jack/47_scenario-switching:
  - added marker events util to add custom lost and found callbacks for each scenario - added 'personal' entry in the ratings personalized data field in data_sources - keeping track of the found product ID in scenario3 - created saveReviewData function in tab-menu to store personal review - calling saveReviewData from 'tilt-forward' callback - changed a few lets to consts
  - added makePanel util
- a few let/const tweaks in sc1

Commit:
Shortened the extra-long test testimonial in the personal data, in preparation for merge into main.

jasonalderman-valtech committed:
Merge pull request #93 from valtech-sd/jack/81_store-review-data

Jack/81 store review data

Commit:
Revert "Added comments on updated build process to the README."

This reverts commit 48c906e4ca171adcb400a289ebfcca19d2f536a0.

Commit:
Built site with new settings.

Commit:
Updated README to reflect fixed npm run build.

jasonalderman-valtech committed:
Merge pull request #92 from valtech-sd/jack/47_scenario-switching

Jack/47 scenario switching

Commit:
Merge branch 'dev' into jack/91_fix-dev-dynamic-images

Commit:
Update build before merge into dev.

jasonalderman-valtech committed:
Merge pull request #94 from valtech-sd/jack/91_fix-dev-dynamic-images

Jack/91 fix dev dynamic images

jasonalderman-valtech reviewed (approved):
Addresses earlier issues. Ready to merge!
